workshops/templates/base.html

```diff
@@ -29,8 +29,12 @@
                 {% navbar_element "Badges" "all_badges" %}
                 {% navbar_element "Airports" "all_airports" %}
                 <li class="divider"></li>
+                {% if perms.workshops.add_person %}
```

Comment:
Instead of showing/not showing the menu element, can you gray out the menu element if permissions aren't there? It's good UI design to have everything always in the same place (muscle memory), just disable the things that people can't get to.